### PR TITLE
[BEAM-6554] Adjust Jenkins directory for spotbugs report

### PR DESCRIPTION
**Pull request by kennknowles**
Spotbugs has a compatible XML format, but puts the output in a different location.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by kennknowles** [comment]
R: @adude3141

File: .test-infra/jenkins/job_PreCommit_Java.groovy [hunk 1]
```diff
@@ -42,7 +42,7 @@ builder.build {
           pattern('**/build/reports/checkstyle/*.xml')
         }
         findBugs {
```
Comment (adude3141) [inline .test-infra/jenkins/job_PreCommit_Java.groovy]: Any reason not to switch to spot bugs here?

Jenkins plugin also has that spotBugs() symbol (see [1], #81)

[1] https://github.com/jenkinsci/warnings-ng-plugin/blob/master/SUPPORTED-FORMATS.md

**Comment by adude3141** [comment]
Run Java PreCommit

**Comment by adude3141** [comment]
Apart from this, reports are available on Jenkins status page.

LGTM. Thx!

**Comment by adude3141** [comment]
Ah, stupid me.... we would need to run seed job first. Currently the 'results' are just the warning 'no files found'

still LGTM, though

**Comment by kennknowles** [comment]
Ah. I didn't realize I could switch `findbugs` to `spotbugs`. I'll do that too.

**Comment by kennknowles** [comment]
run seed job

**Comment by kennknowles** [comment]
`ERROR: Found multiple extensions which provide method spotBugs with arguments [job_PreCommit_Java$_run_closure1$_closure2$_closure3$_closure5$_closure7@390d42b]: [[io.jenkins.plugins.analysis.warnings.FindBugs, io.jenkins.plugins.analysis.warnings.SpotBugs]]`

**Comment by kennknowles** [comment]
Let's switch back to restore the signal and see if we care to fix that issue.

**Comment by kennknowles** [comment]
run seed job

**Comment by kennknowles** [comment]
run java precommit

**Comment by adude3141** [comment]
Well, at least know I know, why you kept that findBugs^^

Sorry for the confusion. Had expected that to simply work. I ll dig into the dsl plugin if I find the time...

**Comment by kennknowles** [comment]
`beam9` disconnected

**Comment by kennknowles** [comment]
run java precommit

**Pull request merged**